[IMP] l10n_vn: SInvoice documentation

kidoodoo opened the pull request:
Recently, a new module released to support integration of Odoo with SInvoice to support e-invoicing for Vietnam.

task - 3631616

afma-odoo reviewed (changes requested):
Thanks @kidoodoo for the update on this page! I added some comments/suggestions. 
In order to harmonize, I think it would be best always to use "S-Invoice" when mentioning the platform, even in Odoo. I know it's often called "SInvoice" or even "Sinvoice", but I think it would avoid misunderstanding. What do you think?
Feel free to ask if you need any clarification ;)

kidoodoo commented on content/applications/finance/fiscal_localizations/vietnam.rst:
```diff
@@ -2,20 +2,201 @@
+
+Invoices can be sent to MyInvois once they have been confirmed. To do so, follow the
+:ref:`invoice sending <accounting/invoice/sending>` steps, and in the :guilabel:`Send` window,
+enable the :guilabel:`Send to Sinvoice` option and click :guilabel:`Send & Print`.
```
@afma-odoo The exact button is 'Send to Sinvoice'.
![image](https://github.com/user-attachments/assets/2a2a64b6-406c-472f-b03b-506bae56f681)

Since all the buttons and info inside Odoo is displayed as "SInvoice", I will convert all "S-Invoice" and "Sinvoice" to "SInvoice"

kidoodoo commented on content/applications/finance/fiscal_localizations/vietnam.rst:
```diff
@@ -2,20 +2,201 @@
+.. note::
+   To create an account, contact :guilabel:`S-Invoice` directly on the portal.
+
+#. Follow the instruction of S-Invoice to choose the invoice template and issue a legal notice of the template chosen.
```
Thank you, I will add the information needed to login and also the steps to add a template and a legal notice.
But I am not so sure as to how much detail this should go into. Does it need to be like how we explain features of Odoo or simply guiding them to the relevant menu suffices?

afma-odoo replied:
@kidoodoo I think the information should be complete enough for the user to complete all the different steps successfully. Does that seem okay to you? ;)

kidoodoo replied:
@afma-odoo Thank you!
One last question on my end on this is that there will be an account manger from SInvoice side and there also is a documentation of SInvoice stating the steps needed (But only in Vietnamese).

Is it something that we could forward to the official documentation of SInvoice or something that we have to still write down the steps or both?

When I refer to the outlook calendar sync doc, it seems to do both, so just want to make sure of this.

Will proceed based on your thoughts! Thank you always!

afma-odoo replied:
@kidoodoo Yes, you can add a "See also" admonition block to add the official documentation, even if it's only in Vietnamese. Here's the info in the documentation guidelines: https://www.odoo.com/documentation/18.0/contributing/documentation/rst_guidelines.html#alert-blocks-admonitions
https://www.odoo.com/documentation/18.0/contributing/documentation/rst_guidelines.html#contributing-rst-hyperlinks
So you can do both.
I'm not sure I understand the "account manager from SInvoice" part. Can you tell me a little bit more about it? Thanks 😊

kidoodoo replied:
@afma-odoo Thank you for the prompt reply!
Will work on them.

For the "Account manager from SInvoice", each account holder of SInvoice has an account manager to support the account holder to be able to use SInvoice portal.

But I think it makes sense we add both like you mentioned above, will work on it accordingly and push it!

afma-odoo replied:
@kidoodoo Thanks for the clarification. Yes, that information is valuable and should be included in the documentation!

Thank you for the update!

kidoodoo commented:
@afma-odoo 

Hello! Hope your day is going well. 
I have pushed an update based on the last review. For the additional part written on the template creation, symbol creation and others, if you think it's better that we have an image or clearer explanation, let me know. Thank you!

afma-odoo reviewed (changes requested):
@kidoodoo Thanks for the updates! I made some suggestions, don't hesitate to ask if you need any additional info ;)

kidoodoo commented on content/applications/finance/fiscal_localizations/vietnam.rst:
```diff
@@ -2,55 +2,311 @@
+   click :guilabel:`Continue`.
+
+   .. Note::
+      The business needs to register for `TB01/AC - Thông báo phát hành hóa đơn` to select it.
```
Hello~
On the government portal, the business should have registered for `TB01/AC - Thông báo phát hành hóa đơn` so that it appears on the selection list to choose from. If the business did not apply, the option will not exist for the user to choose.

I have updated the explanation accordingly, thank you

kidoodoo commented on content/applications/finance/fiscal_localizations/vietnam.rst:
```diff
@@ -2,55 +2,311 @@
+   Choose the :guilabel:`Tax authority receiving the notification`, and click
+   :guilabel:`Complete declaration`. Use USB token for authentication and click
+   :guilabel:`Sign and submit`.
+#. Go to :guilabel:`Declaration` under :guilabel:`Search` menu to select the declaration for
+   `TTB01/AC - Thông báo phát hành hóa đơn`, and click :guilabel:`Search`. Find the declaration
+   file submitted and upload the sample in :file:`.doc` or :file:`.docx` format, and click
+   :guilabel:`Submit Declaration`.
```
For this one hmm, `TTB01/AC - Thông báo phát hành hóa đơn` is the specific one they need to choose for the invoice issuance actually. In that case, wouldn't literals make more sense here?

Others, I will rectify accordingly!

kidoodoo commented:
@afma-odoo 

Hello! Sorry it took a while for the update, had other tasks piled up on my side. I updated based on your latest comment, feel free to let me know after checking!

Merry Christmas in advance and have a great day!

kidoodoo commented:
@afma-odoo 

Hello again and sorry for the additional adjustment. A new method has been found inside SInvoice to issue a invoice issuance notice and I have updated the document accordingly as well.

Rest of the section remains the same, the major change only took place for "Legal notice issuance" which is now "Invoice issuance notice". Sorry for the inconvenience caused, please let me know for anything else needed on my side!

Have a great end of the year again!

afma-odoo commented on content/applications/finance/fiscal_localizations/vietnam.rst:
```diff
@@ -2,55 +2,316 @@
+  - Click :guilabel:`Select the invoice type for issuance`, and then select and fill in the
+    required information below:
+
+    - :guilabel:`Invoice type`: The invoice type to declare an issuance notice on
+    - :guilabel:`Invoice template`: Select from the list of items available based on the invoice
+      type
+    - :guilabel:`Symbol`: Select from the list of items available based on the invoice type
+    - :guilabel:`Quantity`: Total number of invoice to issue for the type selected. Based on the
+      type and template selected, the field is filled in automatically but manually adjustable
+    - :guilabel:`Start date of use`: The date from which the invoice template, range and quantity
+      are used with the notice issued in active status
```
```suggestion
  - :guilabel:`Invoice type`: The invoice type on which to declare an issuance notice.
  - :guilabel:`Invoice template`: Select from the list of items available based on the invoice
    type.
  - :guilabel:`Symbol`: Select from the list of items available based on the invoice type.
  - :guilabel:`Quantity`: Total number of invoices to issue for the type selected. Based on the
    type and template selected, the field is filled in automatically but manually adjustable.
  - :guilabel:`Start date of use`: The date from which the invoice template, range, and quantity
    are used with the notice issued in active status.
```

afma-odoo commented on content/applications/finance/fiscal_localizations/vietnam.rst:
```diff
@@ -2,55 +2,316 @@
+    - :guilabel:`Start date of use`: The date from which the invoice template, range and quantity
+      are used with the notice issued in active status
+
+  - Click :guilabel:`Save` and select more invoice type if necessary by repeating the step above.
+    Once all invoice types to declare are included, click :guilabel:`Save` to finish drafting of
+    the notice. Once ready, click :guilabel:`Send to tax authorities` for an approval. After the
+    approval, :guilabel:`Status` of the notice is changed to :guilabel:`Active`.
```
```suggestion
- Click :guilabel:`Save` and select more invoice types if necessary by repeating the steps above.
  Then click :guilabel:`Save` to finish drafting the notice. 
- Click :guilabel:`Send to tax authorities` for approval. When it's approved, the notice's
 :guilabel:`Status` is changed to :guilabel:`Active`.
```

afma-odoo commented on content/applications/finance/fiscal_localizations/vietnam.rst:
```diff
@@ -2,55 +2,316 @@
+Company configuration
+---------------------
+
+To configure your company information, go to the :guilabel:`Contacts` app, search for your company,
+and select it. Then configure the following fields:
```
```suggestion
To configure your company information, go to the Contacts app, search for your company, and select
 it. Then configure the following fields:
```

afma-odoo commented on content/applications/finance/fiscal_localizations/vietnam.rst:
```diff
@@ -2,55 +2,316 @@
+On the left side of the main screen, in the :guilabel:`Release management` menu, click
+:guilabel:`Invoice symbol` and follow these steps:
+
+  - Click :guilabel:`Add new`, select the :guilabel:`Invoice template`, set the :guilabel:`Status`
+    to :guilabel:`Active` to activate the symbol, and fill in the :guilabel:`Invoice symbol`.
+  - Check :guilabel:`Stop automatic sending to tax authorities` checkbox and
+    :guilabel:`Default for built-in API` checkbox based on preference.
+  - Click :guilabel:`Save` to save.
```
```suggestion
- Click :guilabel:`Add new`, select the :guilabel:`Invoice template`, set the :guilabel:`Status`
  to :guilabel:`Active` to activate the symbol, and fill in the :guilabel:`Invoice symbol`.
- Enable :guilabel:`Stop automatic sending to tax authorities` and
  :guilabel:`Default for built-in API` options based on preference.
- Click :guilabel:`Save`.
```

afma-odoo commented on content/applications/finance/fiscal_localizations/vietnam.rst:
```diff
@@ -2,55 +2,316 @@
+On the left side of the main screen, in the :guilabel:`Release management` menu, click
+:guilabel:`Create issuance notice` and follow these steps:
+
+  - Click :guilabel:`Add new`, select the
+    :guilabel:`Name of the business unit to issue an e-invoice` and the
+    :guilabel:`Tax agency name`. Based on the unit and tax agency selected, :guilabel:`Tax code`,
+    :guilabel:`Address`, :guilabel:`Phone number` and :guilabel:`Separator used` automatically fill
+    in and are ineditable.
```
```suggestion
- Click :guilabel:`Add new`, select the
  :guilabel:`Name of the business unit to issue an e-invoice` and the
  :guilabel:`Tax agency name`. Based on the business unit and tax agency selected, 
  :guilabel:`Tax code`, :guilabel:`Address`, :guilabel:`Phone number` and 
  :guilabel:`Separator used` automatically fill in and are uneditable.
```

afma-odoo commented on content/applications/finance/fiscal_localizations/vietnam.rst:
```diff
@@ -2,55 +2,316 @@
+    :guilabel:`Tax agency name`. Based on the unit and tax agency selected, :guilabel:`Tax code`,
+    :guilabel:`Address`, :guilabel:`Phone number` and :guilabel:`Separator used` automatically fill
+    in and are ineditable.
+  - Click :guilabel:`Select the invoice type for issuance`, and then select and fill in the
+    required information below:
```
```suggestion
- Click :guilabel:`Select the invoice type for issuance`, and then select and fill in the
  required information below:
```

kidoodoo commented:
@afma-odoo 

Hello! Happy belated new year for you too! Wish you and your family all the best for the year ahead ☺️
Sorry for taking such a long time, was busy for the past few weeks. I made the changes based on your review, thank you so much and have a great rest of the week!

afma-odoo commented on content/applications/finance/fiscal_localizations/vietnam.rst:
```diff
@@ -2,55 +2,318 @@
+the bank account to activate QR banking. Fill in the :guilabel:`Bank Identifier Code` specific to
+the bank. Set the :guilabel:`Proxy Type` and fill in the :guilabel:`Proxy Value` field depending on
+your chosen type. The :guilabel:`Proxy Type` depends on the information used to identify the
+:guilabel:`Merchant Account`, such as the card number and bank account number.
```
```suggestion
:guilabel:`Merchant Account`, such as the card number and bank account numbers.
```

afma-odoo commented on content/applications/finance/fiscal_localizations/vietnam.rst:
```diff
@@ -2,55 +2,318 @@
+  to :guilabel:`Active` to activate the symbol, and fill in the :guilabel:`Invoice symbol`.
+- Enable :guilabel:`Stop automatic sending to tax authorities` and
+  :guilabel:`Default for built-in API` options based on preference.
+- Click :guilabel:`Save` to save.
```
```suggestion
- Click :guilabel:`Save`.
```

afma-odoo reviewed (changes requested):
@kidoodoo Thank you for the best wishes ☺️ And thank you for the review! 
I see that there are 2 commits (maybe related to the problem you had pushing them). Can you "squash" them so there's only one left?
I also had 2 minor typos.
Thank you very much!

kidoodoo commented:
@afma-odoo 

Thank you! I squashed the separate commits into one and fixed the typos you mentioned.
Have a great week ahead and let me know for further changes needed!

afma-odoo reviewed (approved):
Thank you @kidoodoo for the updates & the commit squash ;) 
I approve it and add the be-doc-review for the next step!
Have a great week too ☺️

auva-odoo reviewed (approved):
Hello @kidoodoo thank you very much for your work! We're currently testing new guidelines for localization documentation pages, and I saw this as a great opportunity to apply those principles and make some structural adjustments. That's why I force-pushed a few changes. Could you please check I didn't miss anything while doing so? Thank you for your patience, we appreciate it very much!

auva-odoo commented:
@robodoo delegate=afma-odoo

kidoodoo commented:
@auva-odoo 
Thank you so much! I checked and no required information seem to have gone missing at all!

Soon, a new person will be in charge of documentation for the features released from the APAC office and I will resume my role to focus on the release of the localization features. Could help confirm if this can be a good reference to refer for the new PIC to use for the upcoming documentations to work on?

Thank you once again, have a great day ahead!

cc: @afma-odoo

afma-odoo commented:
Hi @kidoodoo! That's great to hear you'll be getting some help with the documentation :)
Thank you so much for checking it! Yes, this could be a good reference for the upcoming documentation, but the new guidelines for localization documentation pages will be more comprehensive. We’ll share them with you as soon as they are ready.
Have a great rest of the week!

@robodoo r+
